plugins/admin/pkg/command/domain/domain.go

```diff
@@ -24,11 +24,14 @@ func NewDomainCmd(p *pkg.AdminParams) *cobra.Command {
 	var domainCmd = &cobra.Command{
 		Use:   "domain",
 		Short: "Manage route domain",
-		Long: `Set default route domain or custom route domain for Service with selectors. For example:
+		Long: `Set default route domain or custom route domain for Service with selectors, also support unset route domain. For example:
```

Comment:
Hi @ZhuangYuZY ,

As the long sentence is hard to read on console, how about we shorten this to 
`Manage default route domain or custom route domain for Service with selectors`
or 
`Set or unset default route domain or custom route domain for Service with selectors`?

Personally I prefer `Manage` as we'll also have a `domain list` later.

Reply:
change is done